Automattic/crowdsignal-ui
# Update Share Modal to include Embed Card settings

## Summary

This PR includes the necessary changes to allow the user to customize the Embed Card settings through the Share Modal

![image](https://user-images.githubusercontent.com/7811225/187561917-2310bd3e-0b13-4af0-95f4-a6e457432583.png)

## Test Instructions

* Open or create a project and make sure it's published
* Click the Share button
* You should be able to see the `Edit card format` button on the Embed Card
* Click the button and play with the settings
* The settings should be saved automatically as soon as you stop editing
